static/src/javascripts/projects/commercial/modules/liveblog-adverts.js

```diff
@@ -121,15 +115,13 @@ export const init = () => {
 		return Promise.resolve();
 	}
 
-	fastdom
+	return fastdom
```

Comment:
fyi this now returns the fastdom promise so we can chain the result in the test. afaics its not chained/awaited anywhere else.